setup.py

```diff
@@ -43,4 +43,8 @@
     ],
     license="GPLv3+",
     python_requires=">=3.6",
+    setup_requires=['setuptools-git-versioning'],
+    version_config={
+        "dev_template": "{tag}.rc{ccount}-git.{sha}",
+    },
```

Comment:
Is the original issue "Set Cachito upstream versions" only for setting the version before a new release?

By using setuptools-git-versioning, should we also remove the `version=`? When releasing a new version, after tagging the new version by `git tag sprintN`, the version can be generated by the tool automatically.

Reply:
Not sure if we'll be setting the version only before release. Version will be used when building images and I'm not sure if that's gonna happen only when we're releasing or every time new commit is merged to master. @chmeliik 

Either way, I'm removing `version=` line.

Reply:
The end goal is that every commit merged to master will trigger an automatic deployment and yeah, building images is part of that.